packages/relic/test/headers/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,280 @@
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:relic/src/headers/headers.dart';
+import 'package:relic/src/relic_server.dart';
+import '../headers_test_utils.dart';
+
+void main() {
+  // Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding
```

Comment:
@SandPod is the reference here enough ?

Reply:
No, I think I was thinking more about references for the specific tests. So for example when you had an empty `Accept-Encoding` header. Where is the source for that decision.